### PR TITLE
Create Releases “2022-04-01-march-2022”

### PR DESCRIPTION
**Pull request by amanifarooque**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by amanifarooque** [comment]
@dawnpm and @MyNameIsMissing - consolidating the process for release notes to GH PRs moving forward. The preview link can be found here: https://cg-a72b65a7-73f0-4859-a1d7-dc953cf6ade8.app.cloud.gov/preview/gsa/search-gov-website/cms/releases/2022-04-01-march-2022/about/updates/releases/march-2022.html

Please let me know any changes you'd like to make!

**Review by MyNameIsMissing — APPROVED** [review]
LGTM

**Comment by amanifarooque** [comment]
@dawnpm URL for Q3 Roadmap: https://cg-a72b65a7-73f0-4859-a1d7-dc953cf6ade8.app.cloud.gov/preview/gsa/search-gov-website/cms/releases/2022-04-01-march-2022/about/updates/roadmap/

File: _releases/2022-04-01-march-2022.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+last_modified_at: April 1, 2022
+sidenav: about_us
+release-year: "2022"
+description: ""
+---
+## Improvements
+
+* **Gem Updates:** We’ve updated gems as needed for a secure environment.
+* **Legacy Code Removal:** We've removed unused code associated with a legacy feature, Search Consumer.
+* **ElasticSearch Upgrade:** We've made progress on updating our system to use ElasticSearch 7.
```
Comment (dawnpm) [inline _releases/2022-04-01-march-2022.md:15]: Please edit to make general, and not mention specific version number thanks

Comment (amanifarooque) [reply _releases/2022-04-01-march-2022.md]: Updated

File: about/updates/roadmap/FY22Q2-roadmap.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+## Overview
+
+At Search.gov we aim to provide an excellent public search experience, a self-service search administration experience, and a rock-solid secure and compliant infrastructure to support it all, while keeping costs down across the government.
+
+We know it’s helpful for our partners at federal agencies to know what’s coming up next, to support planning for your own web platforms. 
+
+This roadmap shows what we are working on now, and what we plan to be working after that. We’ll update this roadmap quarterly.
+
+
+## FY22, Q2: January to March 2022
```
Comment (dawnpm) [inline about/updates/roadmap/FY22Q2-roadmap.md:25]: GSA style says this needs to be "FY 22" with a space. So we should probably make the whole thing "FY 22 Q2" to match the title

File: about/updates/roadmap/FY22Q2-roadmap.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+### Now
+
+* **Faceted Search: User Discovery**
+  * We will be conducting focus groups and initial research to feed into our development plans for faceted search.
+* **Version Upgrades**
+  * We will be upgrading our Rails, ElasticSearch, and Ruby versions to keep them up-to-date and compliant. 
+* **Continuity of Operations Plan Improvements** 
+  * We are taking steps to bolster our system's resiliance and reliability.
+* **New Developer Onboarding**
+  * We are onboarding additional developers in support of our FY22 upcoming projects
```
Comment (dawnpm) [inline about/updates/roadmap/FY22Q2-roadmap.md:36]: Needs to be "FY 22" (though I agree "FY22" is better)

File: about/updates/roadmap/FY22Q2-roadmap.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+* **Results Page Redesign Rollout**
+* **Develop Faceted Search**
+* **Indexing Improvements**
+* **Security and privacy improvements**.
+
+If you have questions or comments, please feel free to [reach out](mailto:search@support.digitalgov.gov) to our team.
+
+## Previous Roadmaps
+
+* [FY22 Q1: October - December 2021]({{ site.baseurl}}/about/updates/roadmap/FY22Q1-roadmap.html)
```
Comment (dawnpm) [inline about/updates/roadmap/FY22Q2-roadmap.md:60]: All these links need to be "FY N"

File: about/updates/roadmap/index.md [hunk 5]
```diff
@@ -22,40 +22,44 @@ We know it’s helpful for our partners at federal agencies to know what’s com
 This roadmap shows what we are working on now, and what we plan to be working after that. We’ll update this roadmap quarterly.
 
 
-## FY22, Q2: January to March 2022
+## FY22, Q3: April to June 2022
```
Comment (dawnpm) [inline about/updates/roadmap/index.md:25]: "FY 22 Q3"

File: about/updates/roadmap/index.md [hunk 6]
```diff
@@ -22,40 +22,44 @@ We know it’s helpful for our partners at federal agencies to know what’s com
...
+  * We are updating some components of our indexing process to ensure resiliency.
 * **Version Upgrades**
-  * We will be upgrading our Rails, ElasticSearch, and Ruby versions to keep them up-to-date and compliant. 
+  * We will be upgrading our ElasticSearch version to keep them up-to-date and compliant. 
 * **Continuity of Operations Plan Improvements** 
   * We are taking steps to bolster our system's resiliance and reliability.
+* **Security and Logging Improvements**
+  * We are adding additional defenses to protect our system from attack, and ways to better visualize incoming traffic. 
 * **New Developer Onboarding**
   * We are onboarding additional developers in support of our FY22 upcoming projects
```
Comment (dawnpm) [inline about/updates/roadmap/index.md:40]: "FY 22"

**Comment by dawnpm** [comment]
Style tweak I missed in past roadmaps, "FY N QN" - here are the other affected files: 
FY22Q1-roadmap
FY21Q4-roadmap 
FY21Q3-roadmap 

These just need the comma removed in the H2, the FY N spacing is already good:
FY21Q2-roadmap
FY21Q1-roadmap
FY20Q4-roadmap

**Review by dawnpm — CHANGES_REQUESTED** [review]
I see you're already hopping on the requested changes, thanks.

**Comment by amanifarooque** [comment]
Changes to FY formatting are now incorporated!

**Review by dawnpm — APPROVED** [review]
Looks great, thanks.

**Pull request merged**